dashboard/src/layout/Header.tsx

```diff
@@ -1,8 +1,20 @@
 import "../App.css"
-
+import SignOutpopup from "./SignOutPopup";
+import PowerSettingsNewOutlinedIcon from '@mui/icons-material/PowerSettingsNewOutlined';
+import { useState } from "react";
 function Header():JSX.Element {
+
+  const [signOut,setSignOut]=useState(false);
+
+  const handleClick=()=>{
+    setSignOut(true);
+  }
+
   return (
     <header className="app-header">
+    {
+          signOut && <SignOutpopup/>
```

Comment:
please use the popup component.

Reply:
@chad11111 for that i would need to install the reactjs-popup dependency do you want me to do this way?

Reply:
No, don't install anything we wrote our own modal.

Have a look here.
https://github.com/komodorio/helm-dashboard/tree/helm-dashboard-v2/dashboard/src/components/modal